scripts/analytics/legacy_wikis.py

```diff
@@ -0,0 +1,97 @@
+# -*- coding: utf-8 -*-
+
+import os
+import sys
+from tabulate import tabulate
+
+from framework.mongo import database
+from website import settings
+from website.addons.wiki import settings as wiki_settings
+
+from .utils import mkdirp
+
+wiki_collection = database['nodewikipage']
+node_collection = database['node']
+
+TAB_PATH = os.path.join(settings.ANALYTICS_PATH, 'tables', 'features')
+mkdirp(TAB_PATH)
+
+def get_wiki_counts():
+
+    query_legacy = {'date': {'$lt': wiki_settings.WIKI_CHANGE_DATE}}
+    wikis_legacy = wiki_collection.find(query_legacy)
+
+    totals = {'pub_proj': {'total': 0, 'total_size': 0, 'largest': 0},
+              'pub_reg': {'total': 0, 'total_size': 0, 'largest': 0},
+              'priv_proj': {'total': 0, 'total_size': 0, 'largest': 0},
+              'priv_reg': {'total': 0, 'total_size': 0, 'largest': 0}}
+
+    metadata = {}
+    for entry in wikis_legacy:
+        node = entry.get('node')
+        page_name = entry.get('page_name')
+        if not node in metadata.keys():
+            node_meta = node_collection.find_one({'_id': node})
```

Comment:
When running this on prod data, this is occasionally returning `None`, which causes a `TypeError` to be raised on the next line. Is this expected?